blueprints/flexberry-edit-form/snippets/flexberry-textbox.hbs

```diff
@@ -6,4 +6,5 @@
     <span style="color:red">{{model.errors.<%= name %>}}</span>
     {{flexberry-textbox placeholder="(no value)" readonly=<%= readonly %> required=true value=model.<%= name %>}}
```

Comment:
"(no value)" should be changed to localized variant when TFS #108286 will complete.